### ISSUE TITLE
Unable to use in Russia and Belarus

### ISSUE THREAD

**Issue by Konev-VA** [shaked6540/YoutubePlaylistDownloader#332]
This issue has been raised before, but it is relevant again.
![image](https://github.com/user-attachments/assets/68482f80-2bd2-4324-8db8-78aed47fcfca)

**Comment by shaked6540** [comment]
Are you using the latest version? Use the one from the  "releases" tab. Remove and re install

**Comment by mangomango** [comment]
Yeah, just run an exe of an unknown Ukrainian origin. What could have gone wrong?

**Comment by Konev-VA** [comment]
> Are you using the latest version? Use the one from the "releases" tab. Remove and re install

It was launched from VS on main branch

**Comment by shaked6540** [comment]
> > Are you using the latest version? Use the one from the "releases" tab. Remove and re install
> 
> It was launched from VS on main branch

Its because youtube explode includes a library that blocks usage from Russia and Belarus, when I build the program I manually remove it. If you use the release version from github it will work
